test/e2e/framework/util.go

```diff
@@ -22,6 +22,9 @@ import (
 	"strings"
 
 	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+
```

Comment:
@tpantelis @skitt @sridhargaddam 

Hi Guys,
Would appreciate your advice here.

I have a linting error for these imports.

I get the following error from "gci":
`test/e2e/framework/util.go:27: File is not `gci`-ed with --skip-generated -s standard,default (gci)`

It actually complains on the empty line 27.
But if I'm removing the empty line, it throws a different error:
```
test/e2e/framework/util.go:26: File is not `goimports`-ed (goimports)
	"k8s.io/apimachinery/pkg/selection"
```
And if I'm running the "goimports", it adds the empty line back.

Could you advice, please?

Reply:
I've run into this before - the linters clash. I fixed it by putting the comment on the same line as the import.

Reply:
Thanks!!